### ISSUE TITLE
Cannot start and some ideas regarding "More"

### ISSUE THREAD

**Issue by xgdgsc** [C2N14/AutomaThemely#5]
plain `automathemely` doesn' t start and shows
```
$ automathemely 
Traceback (most recent call last):
  File "/usr/local/bin/automathemely", line 11, in <module>
    load_entry_point('AutomaThemely==1.2', 'console_scripts', 'automathemely')()
  File "/usr/local/lib/python3.6/dist-packages/automathemely/bin/run.py", line 74, in main
    notify_print_exit('No valid times file found, creating one...', enabled=n_enabled, exit_after=False)
  File "/usr/local/lib/python3.6/dist-packages/automathemely/bin/run.py", line 23, in notify_print_exit
    notify(message)
  File "/usr/local/lib/python3.6/dist-packages/automathemely/__init__.py", line 37, in notify
    raise e
  File "/usr/local/lib/python3.6/dist-packages/automathemely/__init__.py", line 34, in notify
    n.show()
GLib.Error: g-dbus-error-quark: GDBus.Error:org.freedesktop.DBus.Error.NoReply: Message recipient disconnected from message bus without replying (4)

```
while `automathemely --manage` starts. 

I' m on kubuntu 18.04. Does it have to run under gnome? I think you could make it a bit more general purpose, maybe without the gnome dependencies. For example, for some IDEs changing theme could be just changing some file at the time. For the "More" tab you could offer 1. a files switching config UI, and 2.  a command execution UI for user defined commands to run at the switching time. I think this would suffice for most users. While the builtin atom and vscode tabs doesn' t seem very useful since they both have theme switching extensions that work very well.

**Comment by C2N14** [comment]
Sorry for taking so long to reply.

Regarding the gnome dependencies, originally I _did_ plan on making it a more general purpose kinda thing, but I looked a lot into how to dynamically change the themes of KDE and maybe Xfce but for the life of me I couldn't find any way to do so, so if you have any suggestions on how to do it I will probably look into making it compatible.

Now, regarding Atom and VSCode, yes I am very aware that there are extensions to do exactly what this does, however the issue that I found was that these extensions run at different times because they have their own ways to gather their run times, so sometimes you would have a theme mismatch which is why I decided to implement them anyways.
